lib/matplotlib/backends/backend_wx.py

```diff
@@ -625,11 +627,11 @@ def gui_repaint(self, drawDC=None):
         drawDC.DrawBitmap(bmp, 0, 0)
         if self._rubberband_rect is not None:
             # Some versions of wx+python don't support numpy.float64 here.
-            x0, y0, x1, y1 = map(int, self._rubberband_rect)
-            drawDC.DrawLineList(
-                [(x0, y0, x1, y0), (x1, y0, x1, y1),
-                 (x0, y0, x0, y1), (x0, y1, x1, y1)],
-                wx.Pen('BLACK', 1, wx.PENSTYLE_SHORT_DASH))
+            x0, y0, x1, y1 = map(round, self._rubberband_rect)
```

Comment:
Earlier it was possible that the left and bottom bounds could be one pixel off as `int` truncates. Using `round` seems to improve the sitation from practical tests.